tst/EnergyPlus/unit/AirflowNetworkHVAC.unit.cc

```diff
@@ -2212,7 +2212,7 @@ TEST_F(EnergyPlusFixture, AirflowNetwork_TestPressureStat)
 
     // Check indoor pressure and mass flow rate
     EXPECT_NEAR(PressureSet, state->afn->AirflowNetworkNodeSimu(3).PZ, 0.0001);
-    EXPECT_NEAR(0.00255337, state->afn->ReliefMassFlowRate, 0.0001);
+    EXPECT_NEAR(0.06551, state->afn->ReliefMassFlowRate, 0.0001);
```

Comment:
I assume since StdRhoAir is now in the fixture that something has changed BEFORE this unit test sets StdRhoAir.